lib/src/models/webhook.dart

```diff
@@ -87,6 +89,15 @@ class WebhookAuthor extends PartialWebhook implements MessageAuthor {
 
   /// Create a new [WebhookAuthor].
   WebhookAuthor({required super.id, required super.manager, required this.avatarHash, required this.username});
+
+  @override
+  CdnAsset? get avatar => avatarHash == null
+      ? null
+      : CdnAsset(
+          client: manager.client,
+          base: HttpRoute()..avatars(id: id.toString()),
+          hash: avatarHash!,
+        );
```

Comment:
should these things be built in models? would be better to leave building CdnAsset out of here

Reply:
Not really, `hash` is part of the model so it can't be extracted. We could move `client` to the manager but then we'd have to have an `avatarFor(Snowflake id, String avatarHash)` method on the manager which feels weird (having to pass an object's property to the manager).

The only thing we could really move out is the `base` route, and I can't think of a place to put that either - the code is already shortened a lot by the extension which allows us to use `..avatars()` instead of `..add(HttpRoutePart(...))`

Reply:
It's similar to the partial getters (e.g `message.channel` instead of `client.channels[message.channelId]`) in that it's a convenient shortcut for something the user would otherwise have to type out.

Reply:
okay